src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.scss

```diff
@@ -0,0 +1,189 @@
+@import 'mixins';
+@import 'colors';
+
+
+::ng-deep .mat-mdc-menu-panel {
+  width: 400px !important;
```

Comment:
Can't we get around without using important? (maybe adding host helps you here!)

Reply:
i also don't like `!important`, but adding a class solved this